### PR TITLE
Mod 10104

### PR DESCRIPTION
**Pull request by chasls2**
## Check List Before Merging

[DropdownItem](https://federal-spending-transparency.atlassian.net/browse/DEV-10104)

### Author
- [ ] Story Book has been rebuilt (new build artifacts in `/docs`)
- [ ] Components have been rebuilt (new build artifacts in `/dist`)

👆These can be done by executing `npm run build` and committing the build artifacts.

- [ ] All new components have been exported in `/index.js` `(if applicable)`
- [ ] Design Review Complete
- [ ] Relevant Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/release_notes_template.md) to document your changes) `(if applicable)`
- [ ] Integration Status Updated on relevant documentation page `(if applicable)`
- [ ] `index.d.ts` updated with new prop types defined for new components. `(if applicable)`

### Reviewer
- [ ] Code review

### CODE REVIEW

**Comment by nick-torres** [comment]
you need to update the version, commit the rebuilt artifacts

**Comment by chasls2** [comment]
Added the rebuild

File: styles/components/_pageHeader.scss [hunk 1]
```diff
@@ -34,6 +34,16 @@ $z-header: 100;
     @include for-tablet-landscape-up {
       flex-wrap: nowrap;
     }
+    hr {
+      display: blocks;
+      background-color: rgba(223, 225, 226, 0.25);
```
Comment (nick-torres) [inline styles/components/_pageHeader.scss:39]: is there a reason that you're using the rgb function here versus a color variable? i'm not saying you should be using a variable here i'm just curious

File: styles/components/_fiscalYearPicker.scss [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
     @include for-tablet-portrait-down {
-      padding: rem(8) rem(10);
+      padding: rem(12) rem(10);
+      max-height: 52px;
     }
     .usa-dt-picker.usda-fy-picker {
-      background: $gray-cool-5;
-      border: solid 2px rgba(26, 68, 128, 0.25);
+      background:$blue-warm-70v;
+      border: solid 2px rgba(223, 225, 226, 0.25);
```
Comment (nick-torres) [inline styles/components/_fiscalYearPicker.scss:17]: same rgb question

Comment (chasls2) [reply styles/components/_fiscalYearPicker.scss]: @nick-torres changed them

**Review by nick-torres — APPROVED** [review]

**Pull request merged**